js/controller.js

```diff
@@ -2505,13 +2506,15 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
 
     showControlBar: function() {
       this.state.controlBarVisible = true;
+      this.renderSkin();
```

Comment:
Do we need a separate `showControlBarAndRender` in order to avoid rendering in other cases where the code doesn't need it?

Reply:
The `renderSkin` function ends up calling `setState` on skin.js. I agree that we probably do want to render though, I'm not sure how the code that called this without rendering was working. It might be worth checking whether the value actually changed though

Reply:
I'll add in a check to see if the value changed

Reply:
@pilievOoyala it still calls all the render methods and has to do some work in order to determine what has changed. Due to the way our player is structured, a render at this level probably ends up executing the render method of all active components.